Bio/motifs/meme.py

```diff
@@ -172,13 +157,23 @@ def __read_datafile(record, handle):
         line = next(handle)
     except StopIteration:
         raise ValueError("Unexpected end of stream: Expected to find line starting with 'DATAFILE'")
-    if not line.startswith('DATAFILE'):
-        raise ValueError("Line does not start with 'DATAFILE':\n%s" % line)
+    if not line.startswith('PRIMARY SEQUENCES'):
```

Comment:
The automated tests haven't finished yet, but by eye this looks like it will now break on old files which have ``DATAFILE`` here?

Reply:
It does indeed break on the old MEME files in our tests. This change needs to handle both old and new smoothly.

Reply:
I am the most experience Graphic Designer ...
 https://fiverr.com/share/4m9wx